### ISSUE TITLE
Title option not working with multiple time pickers

### ISSUE THREAD

**Issue by wrburgess** [ericjgagnon/wickedpicker#21]
Example of issue here: http://codepen.io/wrburgess/pen/QEVPzk  

**Problem**: Two wickedpickers are on a page. Options for timepicker1 has a custom title. Options for timepicker2 has a custom title.  

Only the custom title for timepicker 1 is rendering.  

**Expected behavior**: Custom titles for timepicker1 and timepicker2 should be rendering  

Am I setting this up wrong or is this a bug?

**Comment by wrburgess** [comment]
cheap and dirty fix, unknown side effects:
`$('.wickedpicker__title').contents().first().replaceWith(this.options.title);`

**Comment by ericjgagnon** [comment]
Yeah the issue is that the timepicker itself is shared across inputs that initialize a wickedpicker.  I didn't want a ton of pickers on the page if there were many instances.

**Comment by wrburgess** [comment]
I completely agree with your approach for that very reason. Having the configuration attach a custom title to the "multiple" pickers isn't that hard with my little patch, but I'm not sure if it's the approach you'd want to take.

**Comment by howardb3115** [comment]
Hello, after looking around at different timepickers, I found that I liked the simplicity of yours and how easy it plugged into my page, once I put all of the pieces in the right places - something your readme should make clearer.  However, I quickly discovered the issue where multiple timepickers all showed the same title, and almost had to abandon your plug-in.  Fortunately, I found this thread and, while using wrburgess' suggested fix didn't work for me, I did find an alternate work around, which I've shown below.

`<body>`
`...`
`<input type="text" id="timepicker-1" class="timepicker" tabindex="1" />`
`<input type="text" id="timepicker-2" class="timepicker" tabindex="2" />`
`<input type="text" id="timepicker-3" class="timepicker" tabindex="3" />`
`...`
`<script>`
`$( '#timepicker-1' ).wickedpicker( { now : "13:00", title : 'Timepicker-1', show : showPicker } );`
`$( '#timepicker-2' ).wickedpicker( { now : "14:00", title : 'Timepicker-2', show : showPicker } );`
`$( '#timepicker-3' ).wickedpicker( { now : "15:00", title : 'Timepicker-3', show : showPicker } );`
`function showPicker( element ) {`
`$( '.wickedpicker__title' ).contents().first().replaceWith( ( ( this.options !== undefined )`
`? this.options.title`
`: this.title ) );`
`}`
`</script>`
`...`
`</body>`

I suspect that the reason that wrburgress' solution didn't work for me might have to do with the way that I'm initializing the options, but in my case I got errors when I tried to create the options as a separate variable object, but nesting the unnamed options object right in the wickedpicker definition worked and implicitly keeps the three timepickers distinct.

**Comment by ericjgagnon** [comment]
Thanks for sticking with my plugin! I haven't released 1.0.0 because I know there are a few issues with it.  I appreciate that you put your solution here and I will try to get this issue resolved over the weekend.

**Comment by howardb3115** [comment]
Thank you for the reply.  I look forward to future developments with your Wickedpicker.Howard Brown-----Original Message-----

**Comment by amitshahc** [comment]
There is no reason to specify any {option} block and pass it to the wickedpicker element if the initial option list is being considered global.  it should be default unless specific option is not passed privately.
